osgp/shared/osgp-throttling-client/src/test/java/org/opensmartgridplatform/throttling/ThrottlingClientTest.java

```diff
@@ -61,16 +61,8 @@ private MockResponse okWithIdResponse(final int id) {
         .setBody(String.valueOf(id));
   }
 
-  @Test
-  void throttlingConfigAndClientAreRegisteredWithTheThrottlingService() {
-    final short throttlingConfigId = 83;
-    final int clientId = 42;
-    this.whenTheThrottlingServiceReturnsIdsOnRegistration(throttlingConfigId, clientId);
```

Comment:
It looks like this was the only place from which private method `whenTheThrottlingServiceReturnsIdsOnRegistration` was called. If it is no longer of use to the test it had better been removed as well.

Reply:
removed method